src/main/java/edu/bu/cs673/stockportfolio/service/portfolio/MarketDataServiceImpl.java

```diff
@@ -59,12 +66,17 @@ public MarketDataServiceImpl(IexCloudConfig iexCloudConfig, RestTemplate restTem
     }
 
     @Override
-    public void doGetSector() {
-        String symbols = "aapl,fb,tsla";
+    public void addNewCompanies(Set<String> symbols) {
```

Comment:
the logic should be separated.

This method should be called something like doGetCompanyNames. Then you should pass the results to a method that adds the company names.

Reply:
Will refactor.